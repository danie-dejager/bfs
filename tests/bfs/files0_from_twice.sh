printf 'basic/c\0' >"$TEST/in1"
printf 'basic/g\0' >"$TEST/in2"
bfs_diff -files0-from "$TEST/in1" -files0-from "$TEST/in2"
