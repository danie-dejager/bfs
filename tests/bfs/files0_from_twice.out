basic/c
basic/c/d
basic/g
basic/g/h
